Slugbotics/ARM
# Added text-to-speech module

Added a text-to-speech module

To test:

0) checkout this branch, install new package by pip installing the requirments.txt

1) make sure tts is enabled in config

2) start the project, see if tts announces startup / shutdown
